test/integration/controllers/offer.spec.js

```diff
@@ -15,19 +15,54 @@ let testOffer = {
   categoryId: '63525e23bf163f5ea609ff2b'
 }
 
+let testOffer2 = {
+  price: 500,
+  proficiencyLevel: 'Beginner',
+  description: 'TEST 123ASD',
+  languages: ['Ukrainian'],
+  subjectId: '63da8767c9ad4c9a0b0eacd3',
+  categoryId: '63525e23bf163f5ea609ff2b'
+}
+
+let testOffer3 = {
+  price: 250,
+  authorRole: 'tutor',
+  proficiencyLevel: 'Beginner',
+  description: 'TEST 123ASD',
+  languages: ['Ukrainian'],
+  subjectId: '63da8767c9ad4c9a0b0eacd3',
+  categoryId: '63525e23bf163f5ea609ff2b'
+}
+
+let testOffer4 = {
+  price: 500,
+  authorRole: 'tutor',
+  proficiencyLevel: 'Beginner',
+  description: 'TEST 123ASD',
+  languages: ['Ukrainian'],
+  subjectId: '63da8767c9ad4c9a0b0eacd3',
+  categoryId: '63525e23bf163f5ea609ff2b'
+}
+
 const updateData = {
   price: 555
 }
 
 describe('Offer controller', () => {
-  let app, server, accessToken, testOfferResponse
+  let app, server, accessToken, testOfferResponse, testOfferResponse2, testOfferResponse3, testOfferResponse4
 
   beforeEach(async () => {
     ;({ app, server } = await serverInit())
     accessToken = await testUserAuthentication(app)
     testOfferResponse = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send(testOffer)
+    testOfferResponse2 = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send(testOffer2)
```

Comment:
do you realy need all responses in each test?

Reply:
Obviously, yes. I require that data to test several use cases.